backbone.ipynb

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
<p>No need for <code>main</code> check  since we're working with a notebook (<code>.ipynb</code>).  This would be more appropriate for a python executable script.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yycho0108/ai604-video-object-pose/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>